utils/HardhatDeploymentUtil.js

```diff
@@ -0,0 +1,1155 @@
+  );
+
+  const erc20TokenExtension = await createERC20Extension(
+    dao,
+    owner,
+    erc20TokenExtFactory,
+    erc20TokenName,
+    erc20TokenSymbol,
+    erc20TokenDecimals,
+    ERC20Extension
+  );
+
+  const executorExtension = await createExecutorExtension(
+    dao,
+    owner,
+    executorExtensionFactory,
+    ExecutorExtension
+  );
+
+  const erc1155TokenExtension = await createERC1155Extension(
+    dao,
+    owner,
+    erc1155TokenExtFactory,
+    ERC1155TokenExtension
+  );
+
+  const extensions = {
+    bank: bankExtension,
+    erc1271: erc1271Extension,
+    nft: nftExtension,
+    erc20Ext: erc20TokenExtension,
+    executorExt: executorExtension,
+    erc1155Ext: erc1155TokenExtension,
+  };
+
+  const { adapters } = await addDefaultAdapters({
+    dao,
+    options,
+    daoFactory,
+  });
+
+  const votingAddress = await dao.getAdapterAddress(sha3("voting"));
+  const votingHelpers = {
+    snapshotProposalContract: null,
+    handleBadReporterAdapter: null,
+    offchainVoting: null,
+    batchVoting: null,
+  };
+
+  if (options.offchainVoting) {
+    const {
+      offchainVoting,
+      handleBadReporterAdapter,
+      snapshotProposalContract,
+    } = await configureOffchainVoting({
+      ...options,
+      dao,
+      daoFactory,
+      votingAddress,
+      bankAddress: extensions.bank.address,
+    });
+    votingHelpers.offchainVoting = offchainVoting;
+    votingHelpers.handleBadReporterAdapter = handleBadReporterAdapter;
+    votingHelpers.snapshotProposalContract = snapshotProposalContract;
+  } else if (options.batchVoting) {
+    const {
+      batchVoting,
+      snapshotProposalContract,
+    } = await configureBatchVoting({
+      ...options,
+      dao,
+      daoFactory,
+    });
+
+    votingHelpers.batchVoting = batchVoting;
+    votingHelpers.snapshotProposalContract = snapshotProposalContract;
+  }
+
+  // deploy test token contracts (for testing convenience)
+  const testContracts = {
+    oltToken: null,
+    testToken1: null,
+    testToken2: null,
+    multicall: null,
+    pixelNFT: null,
+    proxToken: null,
+    erc1155TestToken: null,
+  };
+
+  if (deployTestTokens) {
+    testContracts.testToken1 = await deployFunction("TestToken1", TestToken1, [1000000]);
+    testContracts.testToken2 = await deployFunction("TestToken2", TestToken2, [1000000]);
+    testContracts.multicall = await deployFunction("Multicall", Multicall);
+    testContracts.pixelNFT = await deployFunction("PixelNFT", PixelNFT, [100]);
+    testContracts.oltToken = await deployFunction("OLToken", OLToken, [
+      toBN("1000000000000000000000000").toString()
+    ]);
+    testContracts.proxToken = await deployFunction("ProxToken", ProxToken);
+    testContracts.erc1155TestToken = await deployFunction("ERC1155TestToken", ERC1155TestToken, [
+      "1155 test token",
+    ]);
+  }
+
+  if (finalize) {
+    await dao.finalizeDao({ from: owner });
+  }
+
+  return {
+    dao: dao,
+    adapters: adapters,
```

Comment:
Codacy has a fix for the issue: [Expected property shorthand.](https://app.codacy.com/gh/openlawteam/tribute-contracts/pullRequest?prid=8819146)
```suggestion
    adapters,
```